Source/WebCore/platform/network/curl/CurlContext.cpp

```diff
@@ -27,7 +27,6 @@
 #include "config.h"
 #include "CurlContext.h"
 
-#if USE(CURL)
```

Comment:
I think we should keep `#if USE(CURL)`. IIRC, I heard that it's a good practice to keep source code compil-able for all ports.